ngraph/core/CMakeLists.txt

```diff
@@ -44,21 +44,14 @@ source_group("include" FILES ${PUBLIC_HEADERS})
 add_library(ngraph ${LIBRARY_SRC} ${PUBLIC_HEADERS} ${IE_SHARED_SRCS})
 
 
-if(COMMAND ie_faster_build)
-    ie_faster_build(ngraph
-        UNITY
-        PCH PRIVATE "src/precomp.hpp"
-    )
-endif()
+ie_faster_build(ngraph
```

Comment:
Why it was originally under "if"?

Reply:
because ngraph could be build w/o IE where this function is located